Weapon parser

howareyouman opened the pull request:
Added new class to parse weapons from text. Added some test of this parser. Weapon library will sooner grow up.

howareyouman committed:
new file:   src/main/java/com/w6/nlp/GetDoerAndVictim.java
	modified:   src/main/java/com/w6/nlp/Parser.java
	modified:   src/main/webapp/WEB-INF/jsp/w6.jsp

howareyouman committed:
modified:   src/main/java/com/w6/nlp/GetDoerAndVictim.java

howareyouman committed:
modified:   src/main/java/com/w6/nlp/GetDoerAndVictim.java
	modified:   src/main/java/com/w6/nlp/Parser.java

howareyouman committed:
modified:   main/java/com/w6/controller/EndpointController.java
	modified:   main/java/com/w6/nlp/GetDoerAndVictim.java
	modified:   main/java/com/w6/nlp/Parser.java
	modified:   main/webapp/WEB-INF/jsp/w6.jsp

howareyouman committed:
new file:   src/main/java/com/w6/data/ObjectsAndSubjects.java
	modified:   src/main/java/com/w6/nlp/GetDoerAndVictim.java
	modified:   src/main/java/com/w6/nlp/Parser.java
	modified:   src/main/resources/violentVerbsDictionary.txt
	new file:   src/test/java/ObjectAndSubjectTest.java

howareyouman committed:
modified:   src/main/java/com/w6/nlp/ViolentVerbsParser.java
	new file:   src/main/webapp/resources/violentVerbsDictionary.txt

howareyouman committed:
JavaScript refactoring

Added coloring the words

howareyouman committed:
Testing changes

-null string mistake
-obj and subj correction

howareyouman committed:
new file:   src/main/java/com/w6/nlp/WeaponParser.java
	new file:   src/main/resources/weapons.txt

howareyouman committed:
modified:   src/main/java/com/w6/nlp/Parser.java
	deleted:    src/main/java/com/w6/nlp/WeaponParser.java
	new file:   src/main/java/com/w6/nlp/WeaponsParser.java
	new file:   src/main/webapp/resources/weapons.txt
	new file:   src/test/java/WeaponsTest.java

towelenee commented on src/main/webapp/resources/weapons.txt:
```diff
@@ -0,0 +1,2 @@
+gun
```
Why this file twice in a repo?

howareyouman replied:
this is for server and local tests

towelenee replied:
Is it possible to have one file instead?

howareyouman replied:
I'll check in a minute

howareyouman replied:
Yes, it is!

howareyouman committed:
modified:   src/main/resources/weapons.txt
	modified:   src/main/webapp/resources/weapons.txt

howareyouman committed:
modified:   src/main/resources/weapons.txt
	deleted:    src/main/webapp/resources/violentVerbsDictionary.txt
	deleted:    src/main/webapp/resources/weapons.txt

towelenee commented on src/main/java/com/w6/nlp/WeaponsParser.java:
```diff
@@ -0,0 +1,94 @@
+       parser = globalParser;
+       try {
+           setWeaponsDictionary();
+       } catch (IOException e) {
```
Delete it, it does nothing

towelenee commented on src/main/java/com/w6/nlp/WeaponsParser.java:
```diff
@@ -0,0 +1,94 @@
+                str.replaceAll("\\s+","");
+                weapons.add(str);
+            }
+        } catch (IOException e){
```
It does nothing as well
